src/App.js

```diff
@@ -14,11 +14,103 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    showButton: true,
```

Comment:
речь не только о кнопке, а о старте твоего приложения
```suggestion
    isStarted: true,
```